config_src/coupled_driver/ocean_model_MOM.F90

```diff
@@ -293,6 +294,8 @@ subroutine ocean_model_init(Ocean_sfc, OS, Time_init, Time_in)
                  "The viscosity of the icebergs",  units="m2 s-1",default=1.0e10)
     call get_param(param_file, mod, "DENSITY_ICEBERGS",  OS%density_iceberg, &
                   "A typical density of icebergs.", units="kg m-3", default=917.0)
+    call get_param(param_file, mod, "Lat_fusion",  OS%Lat_fusion, &
```

Comment:
Turns out we already have this parameter called LATENT_HEAT_FUSION so I'm using that...